<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BJ01NN1" failures="1" tests="1" name="Failure Test" time="0.004" errors="0" timestamp="2020-02-18T17:18:54 EST">
  <testcase classname="testcases.TestFailure" name="doLogin" time="0.002">
    <failure type="java.lang.AssertionError" message="Failing the login test">
      <![CDATA[java.lang.AssertionError: Failing the login test
at testcases.TestFailure.doLogin(TestFailure.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- Failure Test -->
